webplugin/js/app/kommunicate-ui.js

```diff
@@ -1904,4 +1897,43 @@ KommunicateUI = {
             }
         );
     },
+    toggleVisibilityOfTextArea: function (assignee,groupMembers) {
+        var isDisableTextAreaEnabled =
+        kommunicate &&
+        kommunicate._globals &&
+        kommunicate._globals.disableTextArea;
+        if(isDisableTextAreaEnabled){
+            if(assignee && groupMembers){
+                for (var i = 0; i < groupMembers.length; i++) {
+                    if (
+                        groupMembers[i].userId === assignee &&
+                        (
+                            groupMembers[i].roleType == '1' ||
+                            groupMembers[i].role == '2' 
+                            )
+                    ) {
+                        kommunicateCommons.modifyClassList(
+                            {
+                                class: ['mck-box-form'],
+                            },
+                            'n-vis',
+                            'vis'
+                        );
+                        break;
```

Comment:
is break required here ?

Reply:
yeah when the widget was loading there comes a case - when it does not go into the if 
and makes the textbox visible - 
can remove it if needed - 
but it won't harm